test/javascript/util/box.js

```diff
@@ -33,8 +33,8 @@ describe('Foundation box', function () {
 
       var dims = Foundation.Box.GetDimensions($("#rect-test"));
 
-      dims.width.should.equal(200);
-      dims.height.should.equal(100);
+      Math.round(dims.width).should.equal(200); // Math.round fix for IE 11 (probably because of rem)
+      Math.round(dims.height).should.equal(100); // Math.round fix for IE 11 (probably because of rem)
```

Comment:
Dimensions are defined in pixels, so why `probably because of rem` ?

Reply:
Take a look at the previous Travis builds. IE11 glitches when we use rem and uses 99.99994... instead of 100. These are rounding bugs that we also see in production on normal websites when we use rem units.

Reply:
When we use rem units on what ? Dimensions here are `height:100px; width:200px;`.

Reply:
I can revert this but still Browserstack reports 99.9999… px instead of 100 so it fails there.

Reply:
See https://travis-ci.org/zurb/foundation-sites/builds/341688519#L3786

Reply:
https://github.com/DURAARK/wiregen/blob/91e9642bc63417d32c9adcdfa5b9e664035cc556/extract_floorplan/EXTRACT.html

Should we really ignore this bug in IE11 or handle it somehow? Either we need a change in the unit test to catch it or find another solution.

Reply:
I'll revert it in about an hour. It seems the unit test was so far not looked at in detail as it was always like this.

https://travis-ci.org/zurb/foundation-sites/builds/326776466#L2881

Reply:
I'll try to reproduce it in our own Browserstack instance with a minimal test. I still gues that it is related to some specific css property as we often see this in production when using Foundation Sites.